input/fsh/omrs-observation.fsh

```diff
@@ -14,6 +14,21 @@ Description: "Observation resource created by a OpenMRS that representing a resu
 * component 0..*
 * encounter 0..1
 * value[x]  0..1
+* partOf 0..*
```

Comment:
Do we support all of these fields in the fhir2 module? Looking at the translator (https://github.com/openmrs/openmrs-module-fhir2/blob/master/api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/ObservationTranslatorImpl.java), it doesn't seem like we set `focus` or `performer` or `note`, for example.

Reply:
Hello @pmanko, We had agreed that we can include fields that even we don't  support and we can mark them with cardinality 0..0, can we go ahead and remove those we dont support.